cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -63,8 +65,9 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
 
   @Override
   public String getPattern() {
-    return String.format("create dataset instance <%s> <%s> [<%s>]",
```

Comment:
it gets a little ambiguous. How do I specify no properties but a description? And no description but properties?